A quick pass of corrections and formatting.

awholegunch opened the pull request:
Fix for https://issues.cask.co/browse/CDAP-4877

A quick pass over the documentation to fix formatting and minor errors.
Standardized on two blank lines before each title and no blank line after a title, to improve readability.

Note: there are two Database-batchsink.md and two Database-batchsource.md:
- core-plugins/docs/Database-batchsink.md
- database-plugins/docs/Database-batchsink.md

and
- core-plugins/docs/Database-batchsource.md
- database-plugins/docs/Database-batchsource.md

Which ones should be deleted?

albertshau commented:
The database ones in core-plugins should be deleted

albertshau commented:
lgtm though there are conflicts

awholegunch commented:
I have resolved the conflict, but please review the last two commits closely.

albertshau commented:
@gokulavasan please take a look at the last commit for the teradata source.  Otherwise lgtm

gsps1 commented on hbase-plugins/docs/HBase-batchsink.md:
```diff
@@ -1,19 +1,20 @@
+
 Properties
 ----------
 **tableName:** The name of the table to read from.
```
there is an issue here, this should be table to write to.. this seems to be copied from batchsource.

gsps1 replied:
also please add a note, saying this table should already exist before running the pipeline.

awholegunch commented:
Please look at the changes in the last commit, as I have addressed both @gokulavasan and some other errors that I found.

albertshau commented:
@gokulavasan still hasn't looked at the teradata changes. Otherwise lgtm

albertshau commented:
o nevermind those changes were pulled in from release/1.2 so i assume they don't need to be reviewed.  lgtm

gokulavasan commented:
LGTM :+1:

Commit:
Fix for https://issues.cask.co/browse/CDAP-4877

A quick pass over the documentation to fix formatting and minor errors.
Standardized on two blank lines before each title and no blank line after a title, to improve readability.

Commit:
Merge branch 'release/1.2' into feature/r1.2_fix_docs

* release/1.2:
  CDAP-3745 remove uneccessary export
  reducing the jar sizes
  Fix teradata documentation to remove references to postgresql

Conflicts:
	database-plugins/docs/Teradata-batchsource.md
